packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -410,30 +419,44 @@ export const TracksTable = ({
       const isLocked = !isFetchingNFTAccess && !hasStreamAccess
       const deleted =
         track.is_delete || track._marked_deleted || !!track.user?.is_deactivated
+      const icon = track.is_unlisted ? (
+        <IconVisibilityHidden color='subdued' size='m' />
+      ) : isContentUSDCPurchaseGated(track.stream_conditions) ? (
+        <IconLock color='subdued' size='m' />
+      ) : isContentCollectibleGated(track.stream_conditions) ? (
+        <IconCollectible color='subdued' size='m' />
+      ) : (
+        <IconSpecialAccess color='subdued' size='m' />
+      )
       return (
-        <div ref={overflowMenuRef}>
-          <OverflowMenuButton
-            className={styles.tableActionButton}
-            isDeleted={deleted}
-            includeEdit={!disabledTrackEdit}
-            includeAlbumPage={!isAlbumPage}
-            includeAddToPlaylist={!isLocked && !track.is_stream_gated}
-            includeFavorite={!isLocked}
-            onRemove={onClickRemove}
-            removeText={removeText}
-            handle={track.handle}
-            trackId={track.track_id}
-            uid={track.uid}
-            date={track.date}
-            isFavorited={track.has_current_user_saved}
-            isOwner={track.owner_id === userId}
-            isOwnerDeactivated={!!track.user?.is_deactivated}
-            isArtistPick={track.user?.artist_pick_track_id === track.track_id}
-            index={cellInfo.row.index}
-            trackTitle={track.name}
-            trackPermalink={track.permalink}
-          />
-        </div>
+        <>
+          {track.is_stream_gated || track.is_unlisted ? (
```

Comment:
super nit: Prefer setting a variable above for this like `showIcon`. Possible that this list of conditions could grow so it would be nicer to have above